platform/ios/src/MGLMapAccessibilityElement.mm

```diff
@@ -52,6 +52,11 @@ - (instancetype)initWithAccessibilityContainer:(id)container feature:(id<MGLFeat
         NSString *nameAttribute = [NSString stringWithFormat:@"name_%@", languageCode];
         NSString *name = [feature attributeForKey:nameAttribute];
 
+        if (name == nil && [feature attributeForKey:@"name"] != nil) {
+            name = [feature attributeForKey:@"name"];
+            languageCode = [feature attributeForKey:@"name_script"];
```

Comment:
This would pass `name_script` into `+[NSOrthography mgl_dominantScriptForMapboxStreetsLanguage:]`, which expects a language code to turn into a script code. But `name_script` is already a script code, so I think we need to pass it directly into `-[NSString(MGLAdditions) mgl_stringByTransliteratingIntoScript:]`.

Reply:
Thanks! Updated to set `dominantScript` rather than `languageCode`.